docs/guidance/assess-against-ids.md

```diff
@@ -1,9 +1,78 @@
 # Evaluating disclosures against the Infrastructure Data Standard
 
+The CoST Infrastructure Data Standard (IDS) is a framework for disclosure which is adapted by CoST national programmes to meet their local needs. This section sets out how to use the **project level schema** and **OCDS** to assess coverage of published data against the IDS. For example, to monitor which elements of IDS are being supplied and whether they are available for all projects or only some.
+
 ```eval_rst
-.. admonition:: Coming soon
+.. admonition:: Note
     :class: Note
 
-    This guidance is currently being developed. Contact data@open-contracting.org for more information in the meantime.
+    .. markdown::
+
+      It is not possible to fully automate checks of whether disclosures from a particular publisher, or disclosures about a particular project, meet the requirements of the CoST IDS. For example, a human check may be needed to determine whether documents linked to from the data contain the required information.
+
+```
+
+## Getting started
+
+*The following steps may require support from a technical expert. You can also contact the OCDS Helpdesk (<a href="mailto:data@open-contracting.org">data@open-contracting.org</a>) for guidance.*
+
+### (1) Check your data formats
+
+First, check that the disclosures you want to analyse are in the correct format. If they are not in the correct format, you will need to convert the data.
+
+#### Project level data
+
+Check whether the project level data is published using the [project level data specification](../projects/index.md)
+
+```eval_rst
+.. admonition:: Tip
+    :class: Tip
+
+    .. markdown::
+
+      You can use a [JSON schema validator](https://json-schema.org/implementations.html#validators) to check that whether your data is published in the correct format.
+
+```
+
+If the data isn’t published using the project level data specification, use the [mapping template](https://docs.google.com/spreadsheets/d/1xHLf_w193pp97zfzhLc_LI-yEXrR_eyscga06Qo1blk/copy) to map the data to the specification and create an OC for Infrastructure JSON file for each project.
+
+<!-- TODO: replace link to google sheets with link to OCP website resources section -->
```

Comment:
Please create an issue (or multiple issues) for the TODOs, which are otherwise less visible.